javasample/src/main/java/com/hanna/test/cookieinformation/javasample/MyApplication.java

```diff
@@ -27,11 +27,24 @@
 import java.util.Map;
 
 import java.util.UUID;
+import kotlin.ParameterName;
+import kotlin.Unit;
 import kotlin.coroutines.Continuation;
+import kotlin.jvm.functions.Function2;
 import kotlinx.coroutines.Dispatchers;
 
 public class MyApplication extends Application implements Consentable {
 
+    @Override public void initSDKWithCompletion(
```

Comment:
If this must be called, what would you say to ask the client to extend a pre-defined application class?

Reply:
@HannahShulman Hello Hannah, good to hear from you. I am not sure I understand 100% the question. We need a single point entry into the SDK in order to make sure we do all the necessary migrations before any other function gets called. But it is up for the discussion whether the init functions should be inside the Consentable interface or not.